.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -12,11 +12,17 @@ body:
     validations:
       required: true
 
+  - type: input
+    id: renovate-version-if-selfhosting
+    attributes:
+      label: If you're self-hosting Renovate, tell us what version of Renovate you run.
+    validations:
+      required: false
+
   - type: dropdown
     id: platform-select
     attributes:
       label: Please select which platform you are using if self-hosting.
-      description: We do not support Renovate on End of Support or End of Life platform versions.
```

Comment:
I've removed the descriptions that were not originally part of the template as requested by @rarkins. I also removed this description, but maybe we want to keep _this_ one?

Reply:
No, let's remove it. I honestly expect some people would lie about their version of server in order that we don't ignore their question. At least without the warning they are more likely to be honest